roles/servicetelemetry/templates/manifest_grafana_ds_prometheus.j2

```diff
@@ -0,0 +1,24 @@
+apiVersion: grafana.integreatly.org/v1beta1
+kind: GrafanaDatasource
+metadata:
+  name: {{ ansible_operator_meta.name }}-ds-stf-prometheus
+  namespace: {{ ansible_operator_meta.namespace }}
+spec:
+  instanceSelector:
+    matchLabels:
+      dashboards: "stf"
+  datasource:
+    name: STFPrometheus
```

Comment:
Also have a hard coded datasource here. We could probably go through and make this slightly more dynamic going forward now that we're managing everything in STO now.

I also found dashboard-linter for Grafana which I'd like to implement at some point here, but the dashboards would need to be updated to have a templated datasource, which is the biggest complaint of the linter.